docs-mslearn/toolkit/hubs/configure-scopes.md

```diff
@@ -16,7 +16,23 @@ ms.reviewer: micflan
 
 Connect FinOps hubs to your billing accounts and subscriptions by configuring Cost Management exports manually or granting FinOps hubs access to manage exports for you.
 
-FinOps hubs use Cost Management exports to import cost data for the billing accounts and subscriptions you want to monitor. You can either configure Cost Management exports manually or grant FinOps hubs access to manage exports for you.
+FinOps hubs use Cost Management exports to import cost data for the billing accounts and subscriptions you want to monitor. **You can configure multiple billing accounts, subscriptions, and even data from other cloud providers** within a single FinOps hub instance. You can either configure Cost Management exports manually or grant FinOps hubs access to manage exports for you.
+
+<br>
+
+## Multi-scope and multi-cloud capabilities
+
+FinOps hubs are designed to handle multiple scopes and even data from multiple cloud providers:
+
+- **Multiple Azure scopes**: You can configure a single FinOps hub to monitor multiple EA billing accounts, MCA billing profiles, subscriptions, and resource groups simultaneously.
+- **Cross-cloud support**: FinOps hubs support the [FinOps Open Cost and Usage Specification (FOCUS)](https://focus.finops.org), which enables ingestion of cost data from other cloud providers like AWS, Google Cloud, and others.
+- **Extensible platform**: The open architecture allows you to extend FinOps hubs to ingest custom data sources beyond standard cloud billing data.
+
+> [!TIP]
+> When configuring multiple scopes, ensure each scope has a unique directory path in your exports to avoid data conflicts. See the [Settings.json scope examples](#settingsjson-scope-examples) section for detailed configuration guidance.
+
+> [!WARNING]
+> Avoid configuring overlapping export scopes as this leads to duplicate cost data. For example, if you configure both an enrollment-level export and a subscription-level export for the same subscription, costs for that subscription will be duplicated in your hub. Always ensure your export scopes are mutually exclusive.
```

Comment:
Use the term "billing account" instead of "enrollment". Enrollment is a legacy term that doesn't apply with MCA.